content/influxdb/v2.0/query-data/execute-queries/data-explorer.md

```diff
@@ -10,20 +12,14 @@ menu:
 influxdb/v2.0/tags: [query]
```

Comment:
Add the following to the frontmatter:

```yml
aliases:
  - /influxdb/v2.0/visualize-data/explore-metrics/
```

Reply:
line 6 won't work? (copied and pasted format from another page, so want to be sure)